test/sanity/validate-modules/validate-modules

```diff
@@ -41,6 +41,14 @@ from schema import doc_schema, option_schema, metadata_schema
 from utils import CaptureStd, parse_yaml
 from voluptuous.humanize import humanize_error
 
+from ansible.compat.six import PY3, with_metaclass
+
+if PY3:
+    # Because there is no ast.TryExcept in Python 3 ast module
+    ast.TryExcept = ast.Try
+    # REPLACER_WINDOWS from ansible.executor.module_common is byte
+    # string but we need unicode for Python 3
+    REPLACER_WINDOWS = REPLACER_WINDOWS.decode('utf-8')
```

Comment:
If I understand why this is being done, and I think I do, the better approach would be instead to open the module file with `rb` instead of changing the type here.

```
diff --git a/test/sanity/validate-modules/validate-modules b/test/sanity/validate-modules/validate-modules
index 6b626f5..f4488d6 100755
--- a/test/sanity/validate-modules/validate-modules
+++ b/test/sanity/validate-modules/validate-modules
@@ -145,7 +145,7 @@ class ModuleValidator(Validator):

         self._python_module_override = False

-        with open(path) as f:
+        with open(path, 'rb') as f:
             self.text = f.read()
         self.length = len(self.text.splitlines())
         try:
```